tailwind.config.js

```diff
@@ -63,7 +63,11 @@ module.exports = {
         'black-800': '#454545',
         'black-900': '#3d3d3d',
         'black-950': '#000000',
+        'active': '#dd4c12',
       },
+      backgroundImage: {
+        checkmark: 'url("/assets/icons/checkIcon.svg")',
+      }
     },
```

Comment:
We already have active color: "accent-600"

Reply:
updated